### DISCUSSION TITLE
球球，救救启动不了,按照文档的demo启动发送验证码验证之后直接报错

### DISCUSSION THREAD

**Discussion by wangshiben** [simple-robot/Discussions#57] [Q&A]
![image](https://user-images.githubusercontent.com/88412139/213169308-f01c5e79-2fea-4cc9-a124-c0ed60a994eb.png)

**Comment by ForteScarlet** [comment]
参考 https://github.com/mamoe/mirai/issues/2418 
可以临时手动更新 `mirai-core` 版本为 [v2.13.3](https://github.com/mamoe/mirai/releases/tag/v2.13.3) ，下个版本也会更新mirai的版本，可留意mirai组件的 [版本更新](https://github.com/simple-robot/simbot-component-mirai/releases) 。
